scripts/install.sh

```diff
@@ -391,10 +391,17 @@ sed_in_place "/.*# avalanche completion/d" $BASH_COMPLETION_MAIN
 echo "source $BASH_COMPLETION_SCRIPT_PATH # avalanche completion" >> $BASH_COMPLETION_MAIN
 if [ $(uname) = Darwin ]
 then
-    BASHRC=~/.bashrc
-    touch $BASHRC
-    sed_in_place "/.*# avalanche completion/d" $BASHRC
-    echo "source $(brew --prefix)/etc/bash_completion # avalanche completion" >> $BASHRC
+    BREW_INSTALLED=true
```

Comment:
I suggest to start with a false value, and then set only if installed. This is a general pattern when evaluating booleans I think: first the case which doesn't assume the value (`BREW_INSTALLED=true` assumes it is installed already, even before we evaluate.)

Reply:
to avoid the script to abort on `which` failure, a `||` connector is used and for that reason the flag is set this way. either way the flag is correctly set.

Reply:
Then a simple rename to `INSTALL_BREW` would be better. This expresses that the installation should be done, and only aborted if it's already installed.